src/libs/validators/index.ts

```diff
@@ -18,7 +18,7 @@ export const validateName = (value: string) => {
 };
 
 export const validateNumber = (value: number) => {
-  const matcher = yup.number().min(1);
+  const matcher = yup.number().min(11);
```

Comment:
while i  have been lazy with this but we think we need to add regx here to match BD mobile number

Reply:
vai backendei jhamela